seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/util/JsonPathProcessorFactory.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.http.util;
+
+import com.jayway.jsonpath.JsonPath;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Factory for creating appropriate JsonPathProcessor instances based on the JsonPath format. */
+public class JsonPathProcessorFactory {
+
+    // List of all available processors
+    private static final List<JsonPathProcessor> PROCESSORS = new ArrayList<>();
+
+    static {
+        // Register all available processors in order of precedence
+        PROCESSORS.add(new ArrayJsonPathProcessor());
+        PROCESSORS.add(new BracketJsonPathProcessor());
+        PROCESSORS.add(new ObjectJsonPathProcessor());
+    }
```

Comment:
If a jsonpath contains three features of `$.store['book'][*].title`, can it still be processed normally? Please add corresponding test cases